<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>史上最萌扫雷小游戏</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="geometry">
     <rect>
      <x>1256</x>
      <y>124</y>
      <width>142</width>
      <height>54</height>
     </rect>
    </property>
    <property name="title">
     <string>新游戏</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>124</y>
      <width>198</width>
      <height>126</height>
     </rect>
    </property>
    <property name="title">
     <string>难易</string>
    </property>
    <addaction name="action_9_9_10"/>
    <addaction name="action_16_16_40"/>
    <addaction name="action_25_25_99"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_9_9_10">
   <property name="text">
    <string>初级 9*9 10个雷</string>
   </property>
  </action>
  <action name="action_16_16_40">
   <property name="text">
    <string>中级 16*16 40个雷</string>
   </property>
  </action>
  <action name="action_25_25_99">
   <property name="text">
    <string>高级 25*25 99个雷</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
